libraries/botbuilder-azure/botbuilder/azure/blob_storage.py

```diff
@@ -0,0 +1,102 @@
+import json
+from typing import Dict, List
+
+from azure.storage.blob import BlockBlobService, Blob, PublicAccess
+from botbuilder.core import Storage, StoreItem
+
+# TODO: sanitize_blob_name
+
+
+class BlobStorageSettings:
+    def __init__(
+        self,
+        container_name: str,
+        account_name: str = "",
+        account_key: str = "",
+        connection_string: str = "",
+    ):
+        self.container_name = container_name
+        self.account_name = account_name
+        self.account_key = account_key
+        self.connection_string = connection_string
+
+
+class BlobStorage(Storage):
+    def __init__(self, settings: BlobStorageSettings):
+        if settings.connection_string:
+            client = BlockBlobService(connection_string=settings.connection_string)
+        elif settings.account_name and settings.account_key:
+            client = BlockBlobService(
+                account_name=settings.account_name, account_key=settings.account_key
+            )
+        else:
+            raise Exception(
+                "Connection string should be provided if there are no account name and key"
+            )
+
+        self.client = client
+        self.settings = settings
+
+    async def read(self, keys: List[str]) -> Dict[str, object]:
+        if not keys:
+            raise Exception("Please provide at least one key to read from storage.")
+
+        self.client.create_container(self.settings.container_name)
```

Comment:
no async lib???

can you open an issue to track the fact that we would like an async version